trustymail/domain.py

```diff
@@ -83,30 +109,42 @@ def get_dmarc_policy(self):
 
 
     def generate_results(self):
+        mail_servers_that_are_listening = [x for x in self.starttls_results.keys() if self.starttls_results[x]["is_listening"]]
```

Comment:
FYI -- Apparently `mail_servers_that_are_listening` isn't actually being used.

Reply:
Good catch, @IanLee1521!  We removed that field from the output yesterday.  I'll create a new pull request for this.

Reply:
Found it while flake8-ifying the code. Will be opening that PR in a bit.

Reply:
Done in https://github.com/dhs-ncats/trustymail/pull/20